scm-ui/ui-api/src/branches.ts

```diff
@@ -111,6 +113,12 @@ export const useBranchDetailsCollection = (repository: Repository, branches: Bra
           return undefined;
         }
         return allPages.length;
+      },
+      onSuccess: data => {
+        const flatMap = data.pages.flatMap(d => d._embedded?.branchDetails);
```

Comment:
```suggestion
        const details = data.pages.flatMap(d => d._embedded?.branchDetails);
```

Reply:
Right, this was just for debugging purposes. I removed the variable completely.